.github/workflows/zeebe-aws-os-dispatch-tests.yml

```diff
@@ -0,0 +1,93 @@
+name: AWS OpenSearch Manual Run Integration Tests
+
+on:
+  workflow_dispatch: { }
+
+defaults:
+  run:
+    # use bash shell by default to ensure pipefail behavior is the default
+    # see https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#exit-codes-and-error-action-preference
+    shell: bash
+
+jobs:
+ integration-tests:
+    name: "[IT] ${{ matrix.name }}"
+    timeout-minutes: 15
+    runs-on: ${{ matrix.runs-on }}
+    strategy:
+      fail-fast: false
+      matrix:
+        group: [ os-exporter ]
+        include:
+          - group: os-exporter
+            name: "Zeebe OS Exporter Integration Tests"
+            maven-modules: "'io.camunda:zeebe-opensearch-exporter'"
+            maven-build-threads: 2
+            maven-test-fork-count: 7
+            runs-on: aws-core-8-default
+    env:
+      ZEEBE_TEST_DOCKER_IMAGE: localhost:5000/camunda/zeebe:current-test
+    services:
+      registry:
+        image: registry:2
+        ports:
+          - 5000:5000
```

Comment:
This setup (providing a locally available temporary Docker registry) is probably redundant now as well since the Docker image build step got removed.